go/libraries/doltcore/sqle/system_variables.go

```diff
@@ -224,7 +224,7 @@ func AddDoltSystemVariables() {
 			Dynamic: true,
```

Comment:
If this is set on server start, should we make Dynamic = false so it can't be reassigned while the engine is running?

I also may not be understanding this change.

Reply:
hmm, would that prevent us from PERSISTing a change to the variable? certainly changing it during a running server isn't going to have any effect, its lifecycle scope is engine startup. but maybe there's no harm letting people write it while the server is running? and perhaps sometimes we'd want to set persist from in a session? In contrast, scanning some of the `Dynamic=false` variables I see things like `admin_port`, `admin_address` that we really don't want users to be able to change

Reply:
Alright, that makes sense.

Reply:
The edit I made below is that by default we do not bootstrap statistics, and unless otherwise specified in the static configuration during sql-server we will bootstrap